### PR TITLE
Add Davey2002_zbMATH01748069 ref + typo + List of mathematical symbols

### PR DESCRIPTION
**Pull request by jamesray1**

### CODE REVIEW

**Comment by pirapira** [comment]
Locally I get
```
! LaTeX Error: Environment tabularx undefined.
```
from `pdflatex`.

**Comment by jamesray1** [comment]
The PR with that in it, I mean.

**Comment by jamesray1** [comment]
Requires #436.

**Comment by jamesray1** [comment]
@pirapira, I tried to resolve this conflict but I think it requires you to resolve them.

File: Paper.tex [hunk 1]
```diff
@@ -106,6 +106,7 @@ \subsection{Previous Work} \label{ch:previous}
 Early work on smart contracts has been done by \cite{szabo1997formalizing} and \cite{miller1997future}. Around the 1990s it became clear that algorithmic enforcement of agreements could become a significant force in human cooperation. Though no specific system was proposed to implement such a system, it was proposed that the future of law would be heavily affected by such systems. In this light, Ethereum may be seen as a general implementation of such a \textit{crypto-law} system.
 
 %E language?
+For a list of terms used in this paper, refer to Appendix \ref{ch:Terminology}.
```
Comment (pirapira) [inline Paper.tex]: `ch:Terminology` is not defined anywhere.

Comment (jamesray1) [reply Paper.tex]: It is now.

**Pull request merged**